packages/core/admin/admin/src/content-manager/pages/ListView/components/TableCells/CellContent.tsx

```diff
@@ -2,79 +2,81 @@ import { Tooltip, Typography } from '@strapi/design-system';
 import isEmpty from 'lodash/isEmpty';
 import styled from 'styled-components';
 
+import { useDoc } from '../../../../hooks/useDocument';
 import { isFieldTypeNumber } from '../../../../utils/fields';
 
 import { CellValue } from './CellValue';
 import { SingleComponent, RepeatableComponent } from './Components';
 import { MediaSingle, MediaMultiple } from './Media';
-import { RelationSingle, RelationMultiple } from './Relations';
 
-import type { FormattedContentTypeLayout } from '../../../../utils/layouts';
-import type { TableHeader } from '../../ListViewPage';
+import type { ListFieldLayout } from '../../../../hooks/useDocumentLayout';
 import type { Attribute, Entity } from '@strapi/types';
 
-interface CellContentProps extends Omit<TableHeader, 'key'> {
+interface CellContentProps extends Omit<ListFieldLayout, 'cellFormatter'> {
   content: Attribute.GetValue<Attribute.Any>;
   rowId: Entity.ID;
-  contentType: FormattedContentTypeLayout;
 }
 
-const CellContent = ({
-  content,
-  fieldSchema,
-  metadatas,
-  name,
-  rowId,
-  contentType,
-}: CellContentProps) => {
-  if (!hasContent(content, metadatas, fieldSchema)) {
+const CellContent = ({ content, mainField, attribute }: CellContentProps) => {
+  const { components } = useDoc();
+
+  if (!hasContent(content, mainField, attribute)) {
     return <Typography textColor="neutral800">-</Typography>;
   }
 
-  switch (fieldSchema.type) {
+  switch (attribute.type) {
     case 'media':
-      if (!fieldSchema.multiple) {
+      if (!attribute.multiple) {
         return <MediaSingle {...content} />;
       }
 
       return <MediaMultiple content={content} />;
 
-    case 'relation': {
-      if (isSingleRelation(fieldSchema.relation)) {
-        return <RelationSingle metadatas={metadatas} content={content} />;
+    /**
+     * TODO: re-add relations to the ListView – tracking issue https://strapi-inc.atlassian.net/browse/CONTENT-2184
```

Comment:
Are we sure links to Jira in the code are okay?

Reply:
For sure, JIRA isn't publicly accessible. What do you think might happen?